website/versioned_docs/version-28.1/GettingStarted.md

```diff
@@ -154,6 +154,25 @@ However, there are some [caveats](https://babeljs.io/docs/en/babel-plugin-transf
 npm install --save-dev ts-jest
 ```
 
+##### Configuration
+
+To configure `ts-jest` for a new project, run:
+
+```bash
+npx ts-jest config:init
```

Comment:
isn't this more for ts-jest repo? 
they have their own docs

Reply:
If this was just a 'related projects' page, I'd agree. But the section is titled "Using Typescript" within "Getting Started". Installing `ts-jest` without the config step doesn't accomplish anything and is more likely to confuse the end users. So it's a hard requirement for "Getting Started"

Reply:
I think we should link to their docs and rather reduce our own if it's currently confusing. `ts-jest` is a separate (if important) project, and I don't wanna have to make sure our docs are up to date with any changes they make